presto-orc/src/main/java/com/facebook/presto/orc/block/SliceDictionaryBlockReader.java

```diff
@@ -111,10 +96,10 @@ public String nextValueAsMapKey()
         int offset = value.getOffset();
         int length = value.length();
         if (writeBinary) {
```

Comment:
I think we can just remove this writeBinary flag, and make both code paths call `VARCHAR.writeSlice(builder, Slices.wrappedBuffer(data, offset, length));` since that's a safe way of writing UTF8 data, and we no longer need to encode varbinary as base64